guides/common/modules/proc_adding-an-scc-account.adoc

```diff
@@ -2,29 +2,35 @@
 = Adding an SCC Account to {Project}
 
 .Prerequisite
-* xref:Installing_the_SCC_Manager_Plugin_{context}[SCC Manager plug-in installed] on your {Project}
+* You have installed the SCC Manager plug-in on your {Project}.
+For more information, see xref:Installing_the_SCC_Manager_Plugin_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content > Content Credentials* and click *Create Content Credential*.
+. Optional: In the {ProjectWebUI}, navigate to *Content > Content Credentials* and click *Create Content Credential*.
 +
 Add the GPG public key for SLES 15 SP3 from https://www.suse.com/support/security/keys/[suse.com].
 . In the {ProjectWebUI}, navigate to *Content > SUSE Subscriptions*.
 . Click *Add SCC Account*.
-. Enter your account name and password.
+. In the *Name* field, enter a name for your SCC account in {Project}.
+. In the *Login* and *Password* fields, enter your SUSE credentials.
+. In the *Base URL* field, you can enter the base URL for the Suse Customer Center.
+By default, it is set to `\https://scc.suse.com`.
 . Optional: Set a *Sync interval* to periodically update the SCC authentication tokens.
 Note that this does not refer to synchronizing content to {Project}.
-. Assign a *GPG key for SUSE products* to the SCC products.
+. Optional: In the *Use GPG key for SUSE products* field, you can select the previously created content credential to automatically add a GPG public key to your SUSE products.
 `zypper` automatically verifies the signatures of each software package to ensure their authenticity.
 +
 [NOTE]
 ====
-You can also set the GPG public key for SUSE repositories at a later stage.
+You can also set the GPG public key for repositories from SUSE at a later stage.
 However, changing it does not affect already synchronized products.
 If you already have synchronized products in {Project}, navigate to *Content > Products* and replace the GPG key in each respective product.
 ====
+. In the *Download Policy* drop down menu, you can select a download policy for your SUSE products.
```

Comment:
```suggestion
. Optional: from the *Download Policy* list, select a download policy for your SUSE products.
```
The word "can" implies that this step is optional, am I right about this?

Reply:
Yes, reworded.